### DISCUSSION TITLE
When can we get H265  for WebRTC

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#3558] [Questions] [answered]
When can we get H265  for WebRTC

**Comment by mgdelafuente** [comment]
The problem with this issue is that the protocol as such does not support it.
https://developer.mozilla.org/en-US/docs/Web/Media/Formats/WebRTC_codecs#supported_video_codecs

**Comment by aler9** [comment, marked as answer]
streaming H265 tracks with WebRTC has been implemented with #4003
